src/main/java/homework_4/singleton/Main.java

```diff
@@ -0,0 +1,18 @@
+package homework_4.singleton;
+
+//Example of using Singleton
+
+public class Main {
+    public static void main(String[] args) {
+
+        Circle myCircle = Circle.getInstance();
+        myCircle.setColor("green");
+
+        Circle myCircle2 = Circle.getInstance();
+        myCircle2.setColor("red");
+
+        System.out.println(myCircle.getColor()); // red
+        System.out.println(myCircle2.getColor()); // red
+        System.out.println(myCircle == myCircle2); // true
```

Comment:
using .equals is much better for objects

Reply:
Objects can be equal, but not the same. Tom's approach is better, I think